src/main/java/uk/gov/hmcts/darts/transcriptions/component/impl/TranscriberTranscriptsQueryImpl.java

```diff
@@ -263,4 +266,69 @@ public List<TranscriberViewSummary> getTranscriberTranscriptions(Integer userId)
         );
     }
 
+    @Override
+    public List<Integer> getAuthorisedCourthouses(Integer userId, Integer roleId) {
+        return jdbcTemplate.queryForList(
+            """
+                select
+                    distinct courthouse.cth_id
+                from
+                    darts.user_account user_account
+                join
+                    (darts.security_group_user_account_ae security_group_user_account_ae
+                join
+                    darts.security_group security_group
+                        on security_group.grp_id=security_group_user_account_ae.grp_id)
+                            on user_account.usr_id=security_group_user_account_ae.usr_id
+                    join
+                        (darts.security_group_courthouse_ae security_group_courthouse_ae
+                    join
+                        darts.courthouse courthouse
+                            on courthouse.cth_id=security_group_courthouse_ae.cth_id)
+                                on security_group.grp_id=security_group_courthouse_ae.grp_id
+                        where
+                            user_account.usr_id=:usr_id
+                            and security_group.rol_id in (:rol_id)
+                """,
+            new MapSqlParameterSource(USR_ID, userId)
+                .addValue(ROL_ID, roleId),
+            Integer.class
+        );
+    }
+
+    @Override
+    public Integer getTranscriptionsCountForCourthouses(List<Integer> courthouseIds, Integer transcriptionStatusId, int userId) {
+        String sql = """
+            SELECT
+                count(*)
+            FROM
+                darts.transcription transcription
+            JOIN
+                darts.court_case court_case
+                    on court_case.cas_id=transcription.cas_id
+            JOIN
+                darts.courthouse courthouse
+                    on courthouse.cth_id=court_case.cth_id
+            INNER JOIN
```

Comment:
Syntactic sugar - but probably worth choosing either `JOIN` or `INNER JOIN` styles rather than a mixture in the same query

Reply:
noted. but will leave as it is for this one.